### PR TITLE
[#26][#33] Implement scrolling with blinking + handle.

### LINKED ISSUE #33: Unable to modify UI components within OnUpdate()
* The FaceTracker is apparently (and obviously) running on a different Thread than AutoscrollActivity, which prevents the FaceTracker from directly modifying UI components.

* Also the CountDownTimer's handler cannot be added to FaceTracker since it does not have a looper implemented.

**Fix on the way**

### PR DESCRIPTION
**Pull request by dansuh17**
Implements #26 
Fixes #33 

**Implements scrolling with blinking**

* Each blink is detected and adjusts scrolling speed.
* Also added a handler to the main UI thread so that FaceTracker thread could send message and modify indirectly the UI components.
* Further calibration on scrolling algorithm needed.

### CODE REVIEW

**Comment by harry2636** [comment]
LGTM. Looks great!

File: Android/toonDra/app/src/main/java/edu/kaist/mskers/toondra/AutoscrollActivity.java [hunk 1]
```diff
@@ -78,75 +98,66 @@ protected void onCreate(Bundle savedInstanceState) {
...
-        break;
-      default:
-        break;
-    }
-    setText();
-    if (scrollVelocity == 0) {
-      return;
-    }
-    countDownTimer = new CountDownTimer(10000 / Math.abs(scrollVelocity), 20) {
+    countDownTimer = new CountDownTimer(MILLIS_IN_FUTURE / Math.abs(scrollVelocity),
```
Comment (heewoonam) [inline Android/toonDra/app/src/main/java/edu/kaist/mskers/toondra/AutoscrollActivity.java:139]: Divide by zero error

File: Android/toonDra/app/src/main/java/edu/kaist/mskers/toondra/AutoscrollActivity.java [hunk 2]
```diff
@@ -46,7 +55,10 @@
   private Button buttonDown;
   private ScrollView scrollView;
   private TextView speedText;
+  private TextView textarea;
```
Comment (heewoonam) [inline Android/toonDra/app/src/main/java/edu/kaist/mskers/toondra/AutoscrollActivity.java]: textarea -> textArea would be better?

**Review by heewoonam — CHANGES_REQUESTED** [review]

**Comment by heewoonam** [comment]
Scrolling accuracy is very bad... Doesn't seem to recognize blinking well enough I guess. Scroll speed varies a lot every time even though when I'm blinking just once

File: Android/toonDra/app/src/main/java/edu/kaist/mskers/toondra/AutoscrollActivity.java (hunk 1 above)
Comment (dansuh17) [reply Android/toonDra/app/src/main/java/edu/kaist/mskers/toondra/AutoscrollActivity.java:139]: done

File: Android/toonDra/app/src/main/java/edu/kaist/mskers/toondra/AutoscrollActivity.java (hunk 2 above)
Comment (dansuh17) [reply Android/toonDra/app/src/main/java/edu/kaist/mskers/toondra/AutoscrollActivity.java]: done

**Comment by dansuh17** [comment]
@heewoonam We'll leave those tasks to future issues :)

**Pull request merged**